src/gpio/mod.rs

```diff
@@ -9,6 +9,8 @@ use ::marker::PhantomData;
 
```

Comment:
Can you please update gpio's mod documentation:

* Describe that in root of gpio module are pins that available on all boards
* Each board's module contains pins that are only available on these particular boards.
* To use these modules, a feature(name of board) should be enabled.